### PR TITLE
update_tmserver: Skip disabled projects by default

### LINKED ISSUE #3919: update_tmserver shouldn't use disabled projects
``update_tmserver`` command shouldn't use disabled projects unless they are specifically requested.

By default we shouldn't use them.  If the user supplies a command line option or uses a specific project name they we should load that project into the TM.

### PR DESCRIPTION
**Pull request by unho**
Fixes #3919.

### CODE REVIEW

**Comment by dwaynebailey** [comment]
lgtm

**Comment by unho** [comment]
Waiting for a second gtm.

File: docs/releases/dev.rst [hunk 1]
```diff
@@ -38,6 +38,8 @@ Details of changes
 - :djadmin:`update_tmserver`:
 
   - Renamed :option:`--overwrite` to :option:`--refresh`.
+  - Translations from disabled project must be explicitly included with
```
Comment (julen) [inline docs/releases/dev.rst]: *projects*

File: docs/server/commands.rst [hunk 2]
```diff
@@ -403,7 +403,8 @@ update_tmserver
 .. versionadded:: 2.7
 
 .. versionchanged:: 2.7.3 Renamed :option:`--overwrite` to :option:`--refresh`.
-   Now it is also possible to import translations from files.
+   Disabled projects translations are no longer added by default. It is also
```
Comment (julen) [inline docs/server/commands.rst]: *projects'*

File: pootle/apps/pootle_app/management/commands/update_tmserver.py [hunk 3]
```diff
@@ -31,6 +31,7 @@ class DBParser(object):
 
     def __init__(self, *args, **kwargs):
         self.INDEX_NAME = kwargs.pop('index', None)
+        self.include_disabled_projects = kwargs.pop('disabled_projs', False)
```
Comment (julen) [inline pootle/apps/pootle_app/management/commands/update_tmserver.py]: What about naming this `disabled_projects`? (FWIW we never use `projs`)

File: pootle/apps/pootle_app/management/commands/update_tmserver.py [hunk 4]
```diff
@@ -43,7 +44,14 @@ def get_units(self, *filenames):
                 'store',
                 'store__translation_project__project',
                 'store__translation_project__language'
-            ).values(
+            )
+
+        if not self.include_disabled_projects:
```
Comment (julen) [inline pootle/apps/pootle_app/management/commands/update_tmserver.py]: I'd suggest to name the attribute `.exclude_disabled_projects`, that way the condition becomes `if self.exclude_disabled_projects:` and the cognitive load for the reader is lower.

**Comment by julen** [comment]
GTM provided the suggestions have been addressed.

**Pull request merged**